content/200-concepts/100-components/02-prisma-client/054-pagination.mdx

```diff
@@ -21,7 +21,7 @@ const results = await prisma.post.findMany({
 })
 ```
 
-![](../../../doc-images/offset-skip-take.png)
+![](/doc-images/offset-skip-take.png)
```

Comment:
Images are not rendering on this page at all: https://deploy-preview-2140--prisma2-docs.netlify.app/docs/concepts/components/prisma-client/pagination

Reply:
Ok, so it works locally. but not in prod. Will revert all these. They should be changed to cloudinary anyway. For another day...